components/script/dom/bluetoothuuid.rs

```diff
@@ -325,22 +337,35 @@ fn resolve_uuid_name(
         prefix: &str)
         -> Fallible<DOMString> {
     match name {
-        // Step 1
+        // Step 1.
```

Comment:
KiChjang wrote the following, when he mentioned, that we should add step anotations.
 ```
I'd like to have step annotations above each Rust statement. 
See examples of step annotations in fn prepare of htmlscriptelement.rs
```
That function uses this style:
https://github.com/szeged/servo/blob/master/components/script/dom/htmlscriptelement.rs#L279